tests/test_apdbCassandra.py

```diff
@@ -110,6 +110,9 @@ class ApdbCassandraTestCase(ApdbCassandraMixin, ApdbTest, unittest.TestCase):
     time_partition_tables = False
     time_partition_start: str | None = None
     time_partition_end: str | None = None
+    # Cassandra stores timestamps with millisecond precision internally,
+    # but pandas seem to convert them to nanosecond type.
+    timestamp_type_name = "datetime64[ns]"
```

Comment:
Should this have been part of "Change default type of timestamp column..."?

Reply:
This is on the same commit as the unit test that uses it, I do not want to squash two commits.